secure_redis/settings.py

```diff
@@ -4,13 +4,19 @@
 from django.core.exceptions import ImproperlyConfigured
 
 
-if hasattr(settings, 'DJANGO_REDIS_SECURE_CACHE_NAME'):
-    cache_name = settings.DJANGO_REDIS_SECURE_CACHE_NAME
-else:
-    cache_name = 'default'
+def get_secure_cache_opts():
+    if hasattr(settings, 'DJANGO_REDIS_SECURE_CACHE_NAME'):
+        cache_name = settings.DJANGO_REDIS_SECURE_CACHE_NAME
+    else:
+        cache_name = 'default'
+    secure_cache_options_settings = settings.CACHES[cache_name]['OPTIONS']
```

Comment:
@GhalebKhaled Will not this cause a `KeyError` if someone uses the library without setting `OPTIONS`.